hadoop-ozone/client/src/main/java/org/apache/hadoop/ozone/client/ObjectStore.java

```diff
@@ -125,10 +128,32 @@ public void createVolume(String volumeName, VolumeArgs volumeArgs)
    * @param bucketName - S3 bucket Name.
    * @throws IOException - On failure, throws an exception like Bucket exists.
    */
-  public void createS3Bucket(String bucketName) throws
-      IOException {
+  public void createS3Bucket(String bucketName) throws IOException {
```

Comment:
Maybe in a separate PR, this can be read once and stored vs. being evaluated each time.

Reply:
Moved config validation and string -> enum conversion to the ObjectStore constructor. Since the client is cached in s3 gateway this will avoid repeated re-evaluation.